typing/typedecl.mli

```diff
@@ -38,6 +38,8 @@ val transl_value_decl:
     Env.t -> Location.t ->
     Parsetree.value_description -> Typedtree.value_description * Env.t
 
+(* If a [Path.t] optional argument is provided, the [Parsetree.type_declaration]
+   argument should satisfy [is_fixed_type] *)
 val transl_with_constraint:
     Env.t -> Ident.t -> Path.t option -> Types.type_declaration ->
```

Comment:
What would you think of switching to an optional argument, for instance `?fixed_row_path:Path.t`?

Reply:
You mean using a meaningful label for an optional argument used once in the whole code base? I buy it.